thirdparty/github.go

```diff
@@ -51,6 +52,15 @@ type GithubPatch struct {
 	CommitMessage  string `bson:"commit_message"`
 }
 
+type SendGithubStatusInput struct {
+	Id    string
+	Owner string
+	Repo  string
+	Ref   string
+	Desc  string
+	Job   string
```

Comment:
Can this be caller instead, since it's not necessarily going to be an amboy job name